apisix/core/request.lua

```diff
@@ -285,4 +285,9 @@ function _M.get_http_version()
     return ngx.req.http_version()
 end
 
+
+function _M.get_method()
```

Comment:
style:

`_M.get_method = ngx.req.get_method` is better style.

Reply:
the `_M.get_http_version` has the same issue, we can fix it in a new PR

Reply:
fixed